Documentation/ApiOverview/UpdateWizards/Creation/Index.rst

```diff
@@ -0,0 +1,74 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-creation-generic:
+
+===============================
+Creating generic update wizards
+===============================
+
+Each update wizard consists of a single PHP file containing a single PHP class. This
+class has to extend :php:`TYPO3\CMS\Install\Updates\AbstractUpdate` and implement the
+abstract methods::
+
+   <?php
+   namespace Vendor\ExtName\Updates;
+
+   use TYPO3\CMS\Install\Updates\AbstractUpdate;
+
+   class ExampleUpdateWizard extends AbstractUpdate
+   {
+       /**
+        * @var string
+        */
+       protected $title = 'Title of this updater';
+
+       /**
+        * Checks whether updates are required.
+        *
+        * @param string &$description The description for the update
+        * @return bool Whether an update is required (TRUE) or not (FALSE)
+        */
+       public function checkForUpdate(&$description)
+       {
+           return true;
+       }
+
+      /**
+       * Performs the accordant updates.
+       *
+       * @param array &$dbQueries Queries done in this update
+       * @param string &$customMessage Custom message
```

Comment:
"Custom message": what for? Do you mean:
"Custom message to be displayed after the update process finished"?